WordPress/Classes/ViewRelated/NUX/UnifiedProloguePages.swift

```diff
@@ -154,54 +159,94 @@ class UnifiedProloguePageViewController: UIViewController {
-                                                         attribute: .height,
-                                                         multiplier: iPadHeightMultiplier,
-                                                         constant: 0)
-    }
-
-    private func setContentViewWidthConstraint() {
-        contentViewWidthConstraint = NSLayoutConstraint(item: contentView,
-                                                        attribute: .width,
-                                                        relatedBy: .equal,
-                                                        toItem: view,
-                                                        attribute: .width,
-                                                        multiplier: 0.7,
-                                                        constant: 0)
-    }
-
-    private func setTitleWidthConstraint() {
-        titleWidthConstraint = NSLayoutConstraint(item: titleLabel,
-                                                  attribute: .width,
-                                                  relatedBy: .equal,
-                                                  toItem: contentView,
-                                                  attribute: .width,
-                                                  multiplier: 1.6,
-                                                  constant: 0)
-        titleWidthConstraint?.priority = .init(999)
-    }
-
-    /// scale factor for the content view on iPad, depending on the orientation
-    private var iPadHeightMultiplier: CGFloat {
-        UIDevice.current.orientation.isPortrait ? 0.4 : 0.5
+    private func activateRegularWidthConstraints() {
+        guard let stackViewAspect = mainStackViewAspectConstraint,
+              let stackViewCenter = mainStackViewCenterAnchor else {
+            return
+        }
+
+        NSLayoutConstraint.activate([stackViewAspect, stackViewCenter])
+    }
+
+    private func activateCompactWidthConstraints() {
+        guard let stackViewLeading = mainStackViewLeadingConstraint,
+              let stackViewTrailing = mainStackViewTrailingConstraint else {
+            return
+        }
+        NSLayoutConstraint.activate([stackViewLeading, stackViewTrailing])
+    }
+
+    private func deactivateRegularWidthConstraints() {
+        guard let stackViewAspect = mainStackViewAspectConstraint,
+              let stackViewCenter = mainStackViewCenterAnchor else {
+            return
+        }
+        NSLayoutConstraint.deactivate([stackViewAspect, stackViewCenter])
+    }
+
+    private func deactivateCompactWidthConstraints() {
+        guard let stackViewLeading = mainStackViewLeadingConstraint,
+              let stackViewTrailing = mainStackViewTrailingConstraint else {
+            return
+        }
+        NSLayoutConstraint.deactivate([stackViewLeading, stackViewTrailing])
+    }
+
+    private func setMainStackViewConstraints() {
+        mainStackViewLeadingConstraint = NSLayoutConstraint(item: mainStackView,
```

Comment:
Would these be a little more concise using layoutAnchors? (mainStackView.leadingAnchor.constraintEqualTo...)